src/sidebar/components/annotation.js

```diff
@@ -471,13 +471,8 @@ function AnnotationController(
   };
 
   this.canFlag = function () {
-    if (session.state.userid === self.annotation.user) {
-      return false;
-    }
-    if (persona.isThirdPartyUser(self.annotation.user, settings.authDomain)) {
-      return true;
-    }
-    return features.flagEnabled('flag_action');
```

Comment:
Good to see another "if is third party user do X, else do Y" branch removed, too